hugr-py/src/hugr/serialization/serial_hugr.py

```diff
@@ -13,8 +11,14 @@
 Port = tuple[NodeIdx, PortOffset | None]
 Edge = tuple[Port, Port]
 
+
+def serialization_version() -> str:
+    """Return the current version of the serialization schema."""
+    return "live"
```

Comment:
Is this really the best we can do? (Sorry - that sounds a bit rude! But I'd hope that it might be, oh, replaced with an actual version number by some part of the release process or something?? That would be fine, but if so I think it'd be good to have a comment here explaining that)

Reply:
I'm not really happy with the unversioned versions, but it's the "no guarantees" value we are currently used. I guess we'll revisit it once `hugr-model` is implemented...